pkg/kv/kvserver/liveness/liveness.go

```diff
@@ -67,6 +67,16 @@ var (
 	ErrEpochAlreadyIncremented = errors.New("epoch already incremented")
 )
 
+type ErrEpochCondFailed struct {
+	expected, actual livenesspb.Liveness
+}
+
+func (e *ErrEpochCondFailed) Error() string {
```

Comment:
Something something implement redaction.

https://github.com/cockroachdb/cockroach/blob/73014328430907dc831374a6517992f4c7c21325/pkg/cli/clierror/cli_error.go#L45-L71

I would also agree to just suppress the logging for this case.

Reply:
There's nothing to redact in this error, the liveness info is not sensitive. Or am I misunderstanding your suggestion?

Reply:
Just to make sure that this error does show up properly in redacted logs. I think right now it will just turn into `<x>` because you're not implementing the redactable printing?

Reply:
Oh right, good point. I'll have a look.